.github/workflows/gh-docker-test.yaml

```diff
@@ -0,0 +1,53 @@
+name: GitHub Docker Test
+env:
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_CLOUD_PROJECT: broad-jade-integration
+  GOOGLE_REGION: us-central1
+  K8_CLUSTER: integration-master
+on:
+  workflow_dispatch: {}
+  pull_request:
+    branches:
+      - fb-dr-1528-gh-docker
+jobs:
+  gh_docker_test:
+    timeout-minutes: 180
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Import Vault Secrets for Development GCR
+        uses: hashicorp/vault-action@v2.1.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.ROLE_ID }}
+          secretId: ${{ secrets.SECRET_ID }}
+          secrets: |
+            secret/dsde/datarepo/dev/sa-key-b64 sa | B64_APPLICATION_CREDENTIALS ;
+      - name: Decode Vault token
+        run: |
+          base64 --decode <<< ${B64_APPLICATION_CREDENTIALS} > ${GOOGLE_APPLICATION_CREDENTIALS}
```

Comment:
Is this necessary? It looks like `setup-gcloud` will set `GOOGLE_APPLICATION_CREDENTIALS` because `export_default_credentials` is `true`.

Reply:
The interactions between the main runner and Docker containers are much more complicated than expected. I added this at an earlier stage to see if I could use these credentials to authenticate inside the Docker containers. Unfortunately this is not possible, which is why a separate configure step is needed. As a result, this step is also no longer needed.